templates/core/venue.html

```diff
@@ -8,7 +8,9 @@
 <section class="venue">
     <section class="venue_info">
         {% if venue_detail.image %}
-        <div class='venue_img' style="background-image: url(https://trello-attachments.s3.amazonaws.com/5786778bdd0206d11e67e444/600f2b5f7af8a413c92bb063/596x397/5951beda3e211f9c244bcf659976cecb/LDN_public_spaces_tab_lobby_expand.jpg)"></div>
+        <div class='venue_img' style="background-image: url({% static 'img/venue_img.png' %});"></div>
```

Comment:
@tsvetelinak0 FYI uploading this static image is OK for now. But when my PR gets approved we'll use a dynamic image url from django - see:
https://github.com/PolyglotDevsLondon/sharespots/pull/116/commits/34380f0f182ae1263101e3f582fa59069b201c2f#diff-e5d658e33229dd81ae3b21f195febada7d398adcbf8cd326da0664ac80e64641R11

Reply:
@lili2311 This is failing Snyk Test and I don't know why. Perhaps Snyk doesn't like binary image files in commits? I've requested access to Snyk for this project. Please can you approve?

Reply:
I uploaded the img as per ticket instructions "Use this image for now for the background". I believe Django will serve venue images from AWS S3 when that is set?

Reply:
OK, this is fine. Images are being done in three phases:

Phase 1: image place holder, what you have done here
Phase 2: image url from Django that's what I've done here.
Phase 3: images from AWS S3, there is a ticket for that here https://trello.com/c/DlcONTae/93-allow-the-admin-to-upload-images-and-store-on-to-s3 and Abu is working on it.

What you have done here is correct for phase 1, well done.